src/Components/Reports/Common/ExecutiveSecondPage.js

```diff
@@ -1,97 +1,52 @@
-import { CheckCircleIconConfig } from '@patternfly/react-icons/dist/js/icons/check-circle-icon';
-import styles from './styles';
-import { DescriptionList } from './DescriptionList';
+import { InstancesTable } from './InstancesTable';
+import propTypes from 'prop-types';
 
-export const ExecutiveSecondPage = () => {
+export const ExecutiveSecondPage = ({ data }) => {
 
-    const stateDescription = [
-        {
-            title: 'Optimized',
-            description: 'Performing at an optimal level',
-            iconPath: CheckCircleIconConfig.svgPath,
-            iconScale: 0.014,
-            fillColor: 'green'
-        },
-        {
-            title: 'Under pressure',
-            description: 'Peaking occasionally',
-            iconPath: TachometerAltIconConfig.svgPath,
-            iconScale: 0.014,
-            fillColor: '#030303'
-        },
-        {
-            title: 'Undersized',
-            description: 'Using more than 80% of system resources',
-            iconPath: AngleDoubleDownIconConfig.svgPath,
-            iconScale: 0.014,
-            fillColor: 'red'
-        },
-        {
-            title: 'Oversized',
-            description: 'Using less than 20% of system resources',
-            iconPath: AngleDoubleUpIconConfig.svgPath,
-            iconScale: 0.014,
-            fillColor: '#f09800'
-        },
-        {
-            title: 'Idling',
-            description: 'Consuming less than 5% of resources',
-            iconPath: AutomationIconConfig.svgPath,
-            iconScale: 0.008,
-            fillColor: '#030303'
-        },
-        {
-            title: 'Waiting for data',
-            description: 'Data has not been received or is being processed. Initial data processing takes up to 24 hours.',
-            iconPath: InProgressIconConfig.svgPath,
-            iconScale: 0.008,
-            fillColor: '#2B9AF3'
-        }
-    ];
+    const { current: currentData, suggested: suggestedData,
+        historical: historicalData } =  data?.instance_types_highlights;  /* eslint-disable-line camelcase */
+    const { stale_count: staleCount } = data?.meta;
 
-    const conditionsDescription = [
+    const instanceTableDetails = [
         {
-            title: 'CPU pressure',
-            description: 'CPU registered peaks higher than 20% over several one-minute time periods'
+            id: 'current_instance_types',
+            heading: 'Most used current instance types',
+            description: 'Based on instances that are already reporting data.',
+            data: currentData
         },
         {
-            title: 'Disk I/O  pressure',
-            description: 'Disk I/O registered peaks higher than 20% over several one-minute time periods'
+            id: 'suggested_instance_types',
+            heading: 'Most suggested instance types (yesterday)',
+            description: 'Yesterday we identified these instance types.',
```

Comment:
Maybe,
`We identified these instance types yesterday.`

Reply:
Please refer [above comment](https://github.com/RedHatInsights/ros-frontend/pull/145#discussion_r1038358462).